packages/ubuntu/debian/postinst

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh -e
+#!/usr/bin/env bash
```

Comment:
Again. Why bash?

Reply:
For the same reason.

By the way, what do you think about replacing
```
#!/bin/sh -e
```
with
```
#!/usr/bin/env (sh|bash)
set -e
```
?

Reply:
I am always using `set -e`. So this is not an issue for me. :)

Reply:
Ok, I think using bash here should be fine for Ubuntu/Debian based distros.

Reply:
I've already changed that back to `sh` after the discussion about `autogen.sh` :)